### PR TITLE
[web] Added news links on front page.

### PR DESCRIPTION
**Pull request by bilke**
Adds links to the 3 recent items in the announcements and the usability category on Discourse.
See the [preview here](https://deploy-preview-2456--ogs.netlify.com).

1. [ ] Feature description was added to the [changelog](https://github.com/ufz/ogs/wiki/Release-notes-6.2.1)

### CODE REVIEW

**Comment by bilke** [comment]
This is ready now.

Please note: icons are not displayed on netlify preview pages in general (due to some license restrictions; we use FontAwesome Pro which you cannot use on arbitrary wild-card domains).

File: web/layouts/index.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {{ define "main" }}
   {{ partial "components/hero" .Params.hero }}
+  <!-- {{ partial "components/quicklinks" .Params.hero }} -->
```
Comment (endJunction) [inline web/layouts/index.html]: could be dropped, I guess...
```suggestion
```

Comment (bilke) [reply web/layouts/index.html]: I would like to refactor / restyle it later, and therefore leave it here.

**Comment by endJunction** [comment]
@Thomas-TK, please review, so it can be merged.

**Comment by Thomas-TK** [comment]
thanks guys that's really a nice and helpful feature

**Pull request merged**